src/components/navbar/Navbar.tsx

```diff
@@ -7,7 +7,7 @@ import ListItem from "@mui/material/ListItem";
 import ListItemButton from "@mui/material/ListItemButton";
 import ListItemIcon from "@mui/material/ListItemIcon";
 import ListItemText from "@mui/material/ListItemText";
-import { mainNavbarItems } from "./NavbarListItems";
+import { mainNavbarItems } from "./navbarListItems";
```

Comment:
```suggestion
import { mainNavbarItems } from "./NavbarListItems";
```

The build fails because you are trying to import from `./navbarListItems` which does not exist. It has to be `./NavbarListItems`